apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -726,6 +742,31 @@ void _wt_onoff_test(void *arg)
 	wm_test_on_off(arg);
 }
 
+int _wt_parse_interop(struct wt_options *opt, int argc, char *argv[])
+{
+	if (argc < 4 || opt == NULL) {
+		return -1;
```

Comment:
_wt_parse_* functions are called from _wt_process() so you don't need to check opt validation.